tests/integration/components/event-table-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('event-table', 'Integration | Component | event table', {
```

Comment:
@CosmicCoder96 see the https://github.com/fossasia/open-event-frontend/blob/development/tests/integration/components/nav-bar-test.js ... See how `moduleForComponent` is defined.